### PR TITLE
Prevent tracks from stealing our watch connectivity session

### PR DESCRIPTION
**Pull request by johnnydhicks**
Ran the app and the delegate methods for our Watch Data Manager are working again!

### CODE REVIEW

**Comment by bjhomer** [comment]
Context: Tracks was unconditionally setting the WCSession's delegate, even though Day One already had one. This was causing a loss of functionality in Day One's watch app.

File: Sources/Model/ObjC/WatchSessionManager.m [hunk 1]
```diff
@@ -48,6 +54,12 @@ - (instancetype)init {
     return self;
 }
 
+- (void)setHasBeenPairedIfPossibleWithSession:(nonnull WCSession *)session {
```
Comment (bjhomer) [inline Sources/Model/ObjC/WatchSessionManager.m]: This is failing the build because `WCSession` is only available on iOS.

**Review by 4sythB — APPROVED** [review]

File: Automattic-Tracks-iOS.podspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'Automattic-Tracks-iOS'
-  s.version       = '0.10.0'
+  s.version       = '0.10.1-beta.1'
```
Comment (diegoreymendez) [inline Automattic-Tracks-iOS.podspec:5]: When the PR is about to get merged we increase the podpsec version, and as soon as it's merged we do a new release using this version number.  This ensures the new version is published in CocoaPods for other Apps to use.

Comment (jleandroperez) [reply Automattic-Tracks-iOS.podspec:5]: We're SO COOL that we're using SPM @diegoreymendez 😎

**Comment by diegoreymendez** [comment]
@johnnydhicks - Since this is all green, I'll go ahead and merge.  Thanks for letting me push this forward!

**Pull request merged**